.htaccess

```
@@ -1,8 +1,20 @@
 <IfModule mod_headers.c>
     Header set Access-Control-Allow-Origin "*"
 </IfModule>
+<IfModule mod_mime.c>
+    AddType font/ttf .ttf
+    AddType font/woff .woff
+    AddType font/woff2 .woff2
+    AddType font/otf .otf
+    AddType application/octet-stream .eot
```

Could do:
```suggestion
    AddType application/vnd.ms-fontobject .eot
```

Per https://www.iana.org/assignments/media-types/application/vnd.ms-fontobject